tests/ui/doc/doc.rs

```diff
@@ -8,7 +8,7 @@
 /// The foo_bar function does _nothing_. See also foo::bar. (note the dot there)
 /// Markdown is _weird_. I mean _really weird_. This \_ is ok. So is `_`. But not Foo::some_fun
 /// which should be reported only once despite being __doubly bad__.
-/// Here be ::a::global:path.
+/// Here be ::a::global:path and _::another::global::path_.
```

Comment:
Could you also add a test where you just have `::` two single colons like this in the text? I think such cases shouldn't be linted, but that's not a clear line. What do you think? :upside_down_face:

Reply:
Agreed, I'll add that.